packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -1636,7 +1636,7 @@ export interface IBasePickerProps<T> extends React.Props<any> {
     itemLimit?: number;
     onBlur?: React.FocusEventHandler<HTMLInputElement | Autofill>;
     onChange?: (items?: T[]) => void;
-    onDismiss?: (ev?: any, selectedItem?: T) => void;
+    onDismiss?: (ev?: any, selectedItem?: T) => boolean | void;

```

Comment:
ITT that this return should be `boolean | undefined` since void isn't really a return value.

Reply:
I wanted to keep this backwards compatible, and it looks like changing void to undefined would require for old onDismiss's to explicitly return undefined (the compiler complains when assigning a function that doesn't return anything to () => void).